install/fedora.md

```diff
@@ -1,5 +1,53 @@
 Installing CJDNS on Fedora
 ==========================
+
+Cjdns was added to the Fedora repository beginning with Fedora 23.  It is in the testing repository until it gains sufficient positive karma.  To install:
```

Comment:
Can we change the sentence so the `cjdns` isn't in the begging.
For example

> Since Fedore 23 cjdns in Fedora's repositories.

Or something like that. It is so there is no confusion how cjdns is(n't) capitalised.